secretmanager/src/main/java/com/example/Quickstart.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example;
+
+// [START secretmanager_quickstart]
+import com.google.cloud.secretmanager.v1beta1.AccessSecretVersionRequest;
+import com.google.cloud.secretmanager.v1beta1.AccessSecretVersionResponse;
+import com.google.cloud.secretmanager.v1beta1.AddSecretVersionRequest;
+import com.google.cloud.secretmanager.v1beta1.CreateSecretRequest;
+import com.google.cloud.secretmanager.v1beta1.ProjectName;
+import com.google.cloud.secretmanager.v1beta1.Replication;
+import com.google.cloud.secretmanager.v1beta1.Secret;
+import com.google.cloud.secretmanager.v1beta1.SecretManagerServiceClient;
+import com.google.cloud.secretmanager.v1beta1.SecretPayload;
+import com.google.cloud.secretmanager.v1beta1.SecretVersion;
+import com.google.protobuf.ByteString;
+
+public class Quickstart {
+
+  public static void main(String... args) throws Exception {
```

Comment:
We have a strong preference to not having CLIs in snippets (most java developers use an IDE, and prefer copy/pasting/running from that) . If you feel strongly about keeping it, use main instead of an overloaded function (else add an overloaded function, as mentioned above).

Reply:
Oh okay - I was following KMS that has executable snippets. Removed.